moto/redshift/models.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, redshift_backend, cluster_identifier, node_type, master_usern
                  preferred_maintenance_window, cluster_parameter_group_name,
                  automated_snapshot_retention_period, port, cluster_version,
                  allow_version_upgrade, number_of_nodes, publicly_accessible,
-                 encrypted, region_name, tags=None):
+                 encrypted, region_name, tags=None, iam_roles_arn=[]):
```

Comment:
Using mutable default arguments can be dangerous; see [here](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments). Probably better to default to None and then set it to an empty list if None after.

Reply:
Ah yes, this definitely needs to be avoided. I've updated the default to `None`.